test/compliment/sources/t_local_bindings.clj

```diff
@@ -57,7 +57,12 @@
                                          '(defn multiarg-fn "docstring"
                                             ([arg] (multiarg-fn arg nil))
                                             ([arg1 arg2] (do-stuff __prefix__))))))
-    => (just ["multiarg-fn" "arg" "arg1" "arg2"] :in-any-order))
+    => (just ["multiarg-fn" "arg" "arg1" "arg2"] :in-any-order)
+
+    (strip-tags (src/candidates "" *ns* (ctx/parse-context
+                                         '(defmethod multimethod "dispatch-val"
+                                            [foo bar & rest] __prefix__))))
+    => (just ["multimethod" "foo" "bar" "rest"]))
```

Comment:
Suggestion: additionally have a `defmethod another-multimethod`, and assert that the completions don't get comflated

Reply:
Thank you for the suggestion. I think it makes more sense to add another dispatch value instead of another multimethod, though I cannot figure out how to do that with current `parse-context` implementation. Could you please provide an example?

Reply:
> Could you please provide an example?

I might PR that at some point, let's merge this one in 👍